### ISSUE TITLE
Tabulator mutators support

### ISSUE THREAD

**Issue by GrishinSergeyLexpharma** [rjaros/kvision#272]
Hi!
Please add support for mutators in columns. http://tabulator.info/docs/4.9/columns
I am currently using "formatterComponentFunction", but it is very slow

**Comment by rjaros** [comment]
If you just want to format the data why don't you use `formatterFunction` instead? It should be as fast (or even faster) as mutators.
But if you really need mutators, it can be done :-)

**Comment by GrishinSergeyLexpharma** [comment]
Thanks!

I tried. Indeed, it works faster if the tags are not returned.

For example, this works quickly
```
formatterFunction = {cell: Tabulator.CellComponent, b: dynamic, onRendered: (callback: () -> Unit) -> Unit ->
                       cell.getValue ()
                    }
```
And this works twice as slow on big data.
```
formatterFunction = {cell: Tabulator.CellComponent, b: dynamic, onRendered: (callback: () -> Unit) -> Unit ->
                      cell.getValue (). let { "<div> $it </div>"  }
                    }
```
The problem with the tabulator is that it re-creates the data inside the cells and rebuilds the tree. 

**Thank you, I will take your advice.**

**Comment by rjaros** [comment]
Released with 4.8.0
